### DISCUSSION TITLE
Can i forbid printing "Line number" in error path?

### DISCUSSION THREAD

**Discussion by lyzliyuzhi91** [CESNET/libyang#2058] [Q&A] [answered]
I used to get error path by ly_errpath in libyang1 to generate netconf error reply, And now I switch to libyang2, and the error path has the extra info "Line number", But I don't want it. So how could i forbid them?

**Comment by michalvasko** [comment, marked as answer]
Sorry but you will have to extract the path from the message, it is not possible to modify its format. It is not designed too well but we would have to increase the SO version to make any changes to it and there is no plan to do that in the near future.

**Comment by lyzliyuzhi91** [comment]
OK. So I create an enhancement issue from this discussion to follow it? I think the extra string operation of "extract" will decrease the performance

**Reply by michalvasko** [reply]
Even if you create a separate issue, it will not make any difference. As I said, new SO version is not planned for now. And I am pretty sure extracting the path will not affect the performance.
